tests/src/res/custom.json

```diff
@@ -19,6 +19,7 @@
   },
 
   "node": {
+    "provider": "http://localhost:9001",
```

Comment:
This property was removed few weeks ago because it's a duplicate of targetHost. So if 'provider' is a better naming, we should simply rename it everywhere to keep only one property.

Reply:
the `provider` address is for the hive node to keep the vault. The `targetHost` is for the hive node to keep the backup data.